codice/ddf
# DDF-2082 Fix MetacardValidityMarkerPlugin so it runs on update requests

#### What does this PR do?
Updates `MetacardValidityMarkerPlugin` so it runs on update requests.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@harrison-tarr 
@bdeining 
@glenhein 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
@kcwire

#### How should this be tested?
Create a validator using a validation JSON file. Ingest a metacard that your validator will fail. Verify that the proper errors and warnings are present on the metacard. Submit an update request for the metacard that will allow your validator to pass it and verify that the updated metacard has no warnings or errors.

Ingest a metacard that your validator will pass. Submit an update request for the metacard that will cause your validator to fail it and verify that the updated metacard has the proper warnings and errors.

Also verify that enforced metacard validators still work (i.e. if you have an enforced metacard validator and it fails your metacard, that metacard is dropped from the create/update operation).

#### What are the relevant tickets?
[DDF-2082](https://codice.atlassian.net/browse/DDF-2082)

#### Checklist:
